judge/judgeapi.py

```diff
@@ -88,6 +88,7 @@ def judge_submission(submission, rejudge, batch_rejudge=False):
             'problem-id': submission.problem.code,
             'language': submission.language.key,
             'source': submission.source.source,
+            'judge_id': judge_id,
```

Comment:
To match `problem-id` in the protocol.

```suggestion
            'judge-id': judge_id,
```